lib/Listener.js

```diff
@@ -174,17 +174,18 @@ class Listener {
     this._consumer.ack(message)
   }
 
-  async _setup ({ queue, event, prefetch = 48 }) {
+  async _setup ({ queue, event, prefetch = 48, subscribe }) {
```

Comment:
Subscribe should default to false here otherwise `exclusive` and `autoDelete` opts would not get set since subscribe will be undefined